internal/host/validator.go

```diff
@@ -1626,3 +1628,21 @@ func (v *validator) noSkipMissingDisk(c *validationContext) (ValidationStatus, s
 	}
 	return ValidationSuccess, successMessage
 }
+
+func (v *validator) serviceCanConnectToSpokeKubeAPI(c *validationContext) (ValidationStatus, string) {
+	// This only applies to scenarios where we have a cluster, kube-api is enabled and the cluster is in "Day2" mode.
```

Comment:
>	// This only applies to scenarios where we have a cluster, kube-api is enabled and the cluster is in "Day2" mode.

That much is apparent from the condition, but this comment doesn't explain why it only applies in that situation